plat/kvm/x86/pagetable64.S

```diff
@@ -121,6 +121,7 @@
  * 0x0000000000000000 - 0x00000000ffffffff Mapping of first 4GB
  *
  * However, the first 1M is inaccessible, except for:
```

Comment:
I now this is not part of this PR, but I am a bit confused by this statement. Shouldn't it be "The first 1G is inaccessible"? Because it seems like we have only 4M accessible in the first GB of the 4GB area

Reply:
the first 1MB may give a GPE when accessing. The others can be read, and possibly written

Reply:
I got it now, my bad